src/components/v2/VaultItemUi/index.tsx

```diff
@@ -0,0 +1,132 @@
+// VaultItemUi
+/** @jsxImportSource @emotion/react */
+import React, { useMemo } from 'react';
+import Paper from '@mui/material/Paper';
+import Typography from '@mui/material/Typography';
+
+import { useTranslation } from 'translation';
+import { formatCommaThousandsPeriodDecimal, formatToReadablePercentage } from 'utilities/common';
+import { Icon } from '../Icon';
+import { Button } from '../Button';
+import { useStyles } from './styles';
+
+type VaultAssetType = 'vai' | 'xvs' | 'vrt';
+
+export interface IVaultItemUiProps {
+  stakeTokenName: VaultAssetType;
+  earnTokenName: VaultAssetType;
```

Comment:
We should use the token IDs here (we don't need to restrict the possible values to just the three we're currently displaying in the desigsns since in the future we might support more vaults with different tokens):
```suggestion
tokenId: TokenId;
rewardTokenId: TokenId;
```